misago/users/signals.py

```diff
@@ -1,11 +1,22 @@
 from django.dispatch import Signal, receiver
-
+from django.contrib.auth import get_user_model
+from misago.core.utils import ANONYMOUS_IP
 
 anonymize_user_content = Signal()
 delete_user_content = Signal()
 username_changed = Signal()
-
+anonymize_old_ips = Signal()
 
 @receiver(username_changed)
 def handle_name_change(sender, **kwargs):
     sender.user_renames.update(changed_by_username=sender.username)
+
+
+UserModel = get_user_model()
```

Comment:
The `UserModel` should be defined before list of signals, and separated by one blank line from it.